jdaviz/app.py

```diff
@@ -984,6 +983,9 @@ def get_subsets(self, subset_name=None, spectral_only=False,
             parent data, return a sky region in addition to pixel region. If
             subset is composite, a sky region for each constituent subset will
             be returned.
+        wcs_from_data : str or None
```

Comment:
can you describe the default here, and how this argument can be used to override the default? if you are pixel linked, you still can return a sky region without setting wcs_from_data